labgrid/driver/networkusbstoragedriver.py

```diff
@@ -49,6 +51,52 @@ def on_activate(self):
     def on_deactivate(self):
         pass
 
+    @step(args=['filenames', 'target_dir', 'partition'])
+    def copy_files(self, filenames, target_dir, partition=1):
+        if not self.storage.path:
+            raise ExecutionError(
+                "{} is not available".format(self.storage_path)
```

Comment:
`self.storage_path` does not exist. This typo was copied from the NetworkUSBStorageDriver. #556 aims to fix it.